DP02_13a_Image_Cutout_SciDemo.ipynb

```diff
@@ -73,11 +73,11 @@
    "source": [
```

Comment:
<p>"the cell also creates" --&gt; "create"? (remember we're supposed to be using imperative mood)</p><p> </p><p>Also, I think "a SpherePoint class" should be "a SpherePoint object"?</p><p> </p><p>"These will be used" --&gt; "Use these"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Thank you!</p>